### DISCUSSION TITLE
motors arming angle max INAV

### DISCUSSION THREAD

**Discussion by FF1970** [iNavFlight/inav#10354] [Ideas]
Hello everyone
Is it possible, as with Betaflight, to manage the motor arming angle up to 180° under INAV?

thanks

**Comment by anti-vaxxer** [comment]
Hi FF1970, 
yes, in CLI
set small_angle = 180
save

**Comment by sensei-hacker** [comment]
Yes and it's normally not what you actually want to do. Do you launch your aircraft upside-down?
If you're carrying your aircraft directly nose up, so the prop is near your face, do you want it to spin up in your face?

Most of the time, when people ask about 180° they really want more like 35°-40°.

**Comment by FF1970** [comment]
ok 
It is for a specific drone 
Thanks for the info I will try
